tools/manage_clvm.py

```diff
@@ -141,7 +143,7 @@ def from_clvm_paths(cls, paths: ClvmPaths, hash_dict: typing.Dict[str, str] = {}
         hex_bytes = paths.hex.read_bytes()
```

Comment:
unrelated to your patch, but if this file contains hex, it would seem more practical to open it in text mode, so `read()` returns a `str` instead of `bytes`.

Reply:
Yes, that seems better. I'll put in another PR.